app/products/calls/alerts.ts

```diff
@@ -98,7 +107,7 @@ const doJoinCall = async (serverUrl: string, channelId: string, isDMorGM: boolea
         return;
     }
 
-    const res = await joinCall(serverUrl, channelId);
+    const res = await joinCall(serverUrl, channelId, user.id);
```

Comment:
this `user.id` can belong to the wrong server, see L#94, is that expected?

Reply:
(please correct me if I'm wrong) Since `doJoinCall` will only be called while the user is in the call's server, this should be the user's correct id. Unless I'm misunderstanding how `getActiveServerDatabase` works?
Basically -- I'm calling `getActiveServerDatabase` here so that I don't have to pass down the userId from the `channelInfo` or `channel` screens.

Reply:
yes, `getActiveServerDatabase` will return the current server database, that is for sure.. my worry here is in case the action is called with a serverUrl other than the current active server. this can be prevented by just getting the correct database for the serverUrl passed in the argument with `DatabaseManager.getServerDatabaseAndOperator` <- this can throw.

Reply:
Ah you're right, that's better, thanks!